Modules/Core/Common/test/itkOctreeTest.cxx

```diff
@@ -74,6 +75,10 @@ itkOctreeTest(int, char *[])
 
   using OctreeType = itk::Octree<unsigned int, 16384, IdentityMap<unsigned int, 16384>>;
   auto octree = OctreeType::New();
+
+  // ITK_EXERCISE_BASIC_OBJECT_METHODS(octree, Octree, OctreeBase);
```

Comment:
I have commented this. It seems to upset Windows because the test is telling that the superclass name is `Object`:
https://dev.azure.com/itkrobotwindow/ITK.Windows/_build/results?buildId=7506&view=logs&j=2d2b3007-3c5c-5840-9bb0-2b1ea49925f3&t=77aad734-2057-5694-9ae2-ee1f5f26eae8&l=55016

which seems not to be the case, `Object` is the superclass of `OctreeBase`.

Both `Linux` and `macOS` seemed to be happy for the very same commit 9ed699b:
https://dev.azure.com/itkrobotlinux/ITK.Linux/_build/results?buildId=7524&view=results
https://dev.azure.com/itkrobotmacos/ITK.macOS/_build/results?buildId=7455&view=results

I do not see anything wrong, and it reminds me of other past cases, e.g. https://github.com/InsightSoftwareConsortium/ITK/pull/2286#issuecomment-778559383

So I preferred to comment the statement, as it seems to deserve a separate investigation.

Might be loosely linked to #3008, although the latter was mainly opened due to the FFT backend name of class and GCC issues.

Reply:
Another one I found yesterday was `itk::StdStreamLogOutput`, whose parent class is `itk::LogOutput`; when trying to exercise basic object methods `ITK_EXERCISE_BASIC_OBJECT_METHODS`, it expects `itk::Object` as the superclass, which is `itk::LogOutput`'s parent. Not sure if all these are somehow related to symbol exportation macros, i.e. `ITKCommon_EXPORT`. Just guessing.